substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/NativeImage.java

```diff
@@ -1203,11 +1204,30 @@ protected int buildImage(List<String> javaArgs, LinkedHashSet<Path> bcp, LinkedH
         }
 
         int exitStatus = 1;
+        File argsFile = null;
         try {
+            if (OS.getCurrent() == OS.WINDOWS) {
```

Comment:
Use `Platform.includedIn(Platform.WINDOWS.class)` instead of `OS.getCurrent() == OS.WINDOWS`. 
`Platform.includedIn(...)` is built-time constant. It ensures the code for special handling is not included in native-image images for Platforms != Windows.

Reply:
Yes, I said the same in the PR description. Is that what you want me to do? Just remove the platform check and make this the default for all platforms?

Reply:
Ok